Backend/controllers/adminc.js

```diff
@@ -0,0 +1,16 @@
+let admin = [];
+const createAdmin = async (req, res) => {
+  const { email, password } = req.body;
+  admin.push(email);
+  //   console.log(email);
+  res.status(201).json({ message: "Admin created successfully", email });
+};
+const getAdminById = async (req, res) => {
+  const id = req.params.adminId;
+  console.log(admin[0]);
+  res.json({ admin: admin[0] });
```

Comment:
Instead of directly sending the response, it would be better if you perform the necessary checks. For example, say the admin id doesn't exist, then an internal server error will be raised.

Reply:
I will add checks for both user and admin as soon as DB is integrated.